test/integration/nested-persistence-test.ts

```diff
@@ -199,14 +224,16 @@ describe('nested persistence', function() {
     beforeEach(function() {
       let genre = new Genre({ name: 'Horror' });
       let book = new Book({ title: 'The Shining', genre: genre });
+      let specialBook = new Book({ title: 'The Stand' });
       instance.books = [book];
+      instance.specialBooks = [specialBook];
     });
 
     // todo test on the way back - id set, attrs updated, isPersisted
     // todo remove #destroy? and just save when markwithpersisted? combo? for ombined payload
     // todo test unique includes/circular relationshio
     it('sends the correct payload', function(done) {
-      instance.save({ with: { books: 'genre' } }).then((response) => {
+      instance.save({ with: { books: 'genre', specialBooks: true } }).then((response) => {
```

Comment:
Super minor, but could we make this `specialBooks: {}`? It better models what we actually expect (and will enforce in TypeScript soon), the `true` just *happens* to work here and these tests can be useful as documentation.

Reply:
Ahh yes - was honestly just a guess on my part 😄  updated!